db/seeds.rb

```diff
@@ -8,70 +10,52 @@ def boolean_check(value)
   end
 end
 
-def null_check(value)
-  if value != "NULL"
-    return value
-  end
-end
-
-def build_class_requirements(spell)
+def build_spell_requirements(spell_data)
   return {
-    alchemist: null_check(spell[:alchemist]),
-    antipaladin: null_check(spell[:antipaladin]),
-    bard: null_check(spell[:bard]),
-    bloodrager: null_check(spell[:bloodrager]),
-    cleric: null_check(spell[:cleric]),
-    druid: null_check(spell[:druid]),
-    hunter: null_check(spell[:hunter]),
-    inquisitor: null_check(spell[:inquisitor]),
-    investigator: null_check(spell[:investigator]),
-    magus: null_check(spell[:magus]),
-    medium: null_check(spell[:medium]),
-    mesmerist: null_check(spell[:mesmerist]),
-    occultist: null_check(spell[:occultist]),
-    oracle: null_check(spell[:oracle]),
-    paladin: null_check(spell[:paladin]),
-    psychic: null_check(spell[:psychic]),
-    ranger: null_check(spell[:ranger]),
-    shaman: null_check(spell[:shaman]),
-    skald: null_check(spell[:skald]),
-    sorcerer: null_check(spell[:sor]),
-    spiritualist: null_check(spell[:spiritualist]),
-    summoner: null_check(spell[:summoner]),
-    witch: null_check(spell[:witch]),
-    wizard: null_check(spell[:wiz])
+    verbal:   boolean_check(spell_data[:verbal]),
+    somatic:  boolean_check(spell_data[:somatic]),
+    material: boolean_check(spell_data[:material]),
+    focus:    boolean_check(spell_data[:focus])
   }
 end
 
-def build_spell_requirements(spell)
-  return {
-    verbal:   boolean_check(spell[:verbal]),
-    somatic:  boolean_check(spell[:somatic]),
-    material: boolean_check(spell[:material]),
-    focus:    boolean_check(spell[:focus])
-  }
+def build_class_requirements(spell_data)
+  PathfinderClass.all.each do |klass|
```

Comment:
I would call this `pathfinder_class`

`klass` is generally reserved for meta programming involving class names

Reply:
Understood, makes sense to preserve that convention.